pkg/model/config/config.go

```diff
@@ -147,44 +220,84 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
-	Group   string `json:"group,omitempty"`
-	Version string `json:"version,omitempty"`
-	Kind    string `json:"kind,omitempty"`
+// simplify omits some fields that can be restored at unmarshalling.
+func (c *Config) simplify() {
+	if c.IsV3() {
```

Comment:
```suggestion
```
Should be done for all project versions.

Reply:
V2 doesnt require simplification because its only storing group version and kind, and none of them are simplified.

Reply:
It cannot be fixed for V3.  So, it tough to be !v2.